src/js/view/dock.js

```diff
@@ -1,365 +1,45 @@
 /*jshint evil:true*/
 define([
-    'view/overlay',
-    'view/touch',
-    'events/events',
+    'handlebars-loader!templates/dock.html',
     'utils/helpers',
-    'utils/css',
-    'underscore'
-], function(Overlay, Touch, events, utils, cssUtils, _) {
+    'utils/underscore'
```

Comment:
Good call on this
On Apr 14, 2015 7:55 PM, "Rob Walch" <notifications@github.com> wrote:

> In src/js/view/dock.js
> <https://github.com/jwplayer/jwplayer/pull/380#discussion_r28383107>:
>
> >      'utils/helpers',
> > -    'utils/css',
> > -    'underscore'
> > -], function(Overlay, Touch, events, utils, cssUtils, _) {
> > +    'utils/underscore'
>
> ...fixed by removing alias so that we don't have issues or conflicts
> between requirejs and webpack 295b739
> <https://github.com/jwplayer/jwplayer/commit/295b739eec48927239aa56b915e1f8f0df5d69a7>
>
> Looks like require wasn't event trying to load 'utils/underscore' because
> it already loaded 'underscore' which was setup to load from the same path
> in require's config.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jwplayer/jwplayer/pull/380/files#r28383107>.
>